recipes/tsase/LICENSE.txt

```diff
@@ -0,0 +1,12 @@
+tsase is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2.1 of the License, or
+(at your option) any later version.
+
+tsase is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with tsase.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
Where did you find this license? I haven't been able to find a source for it from a quick search.

Also, a copy of the license shuld be included.